pkg/scripts/bigchaindb-monit-config

```diff
@@ -93,7 +93,7 @@ case \$1 in
   start_bigchaindb)
 
     pushd \$4
-      nohup bigchaindb -l DEBUG start >> \$3/bigchaindb.out.log 2>> \$3/bigchaindb.err.log &
+      nohup bigchaindb -l DEBUG start > /dev/null 2>&1 &
```

Comment:
Why set logging level to debug? I think its better to leave logging flag out of the command here and let it pick up logging configs from the config file.

Reply:
It is a test network, but I can change if that is necessary.

Reply:
Some people might use our Monit script for their production node.